style.css

```diff
@@ -760,7 +760,6 @@ input[type=submit]:disabled {
 
 .open {
   height: 100vh;
-  
 }
 
 @media only screen and (min-width: 768px) {
```

Comment:
- [ ] Please let's fix this issue to make your project nice, currently, if I click the hamburger icon and the mobile menu appears, after the mobile menu appears and the screen width is adjusted to the desktop screen, the mobile menu should automatically disappear to enhance user experience :+1: but it is not mobile menu is not disappearing as you can see below after a adjust the screen with to desktop so make sure that you fix this :pray: 

  [Screencast from 05-15-2023 07:31:52 PM.webm](https://github.com/YoutLone/Portfolio/assets/92303891/fe08e094-d4c2-4cba-89e5-b26f74463e58)